### PR TITLE
#14705 Fixed EditMavenArtifactDialog logic

### PR DESCRIPTION
**Pull request by Destrolaric**

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/EditMavenArtifactDialog.java [hunk 1]
```diff
@@ -108,6 +108,7 @@ public void widgetSelected(SelectionEvent e) {
                     //check if we are on raw tab. When trues to parse
                     if (tabFolder.getSelectionIndex() == 0 && tabFolder.getItemCount() != 1){
                         UIUtils.asyncExec(EditMavenArtifactDialog.this::parseArtifactText);
+
```
Comment (flambchap) [inline plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/EditMavenArtifactDialog.java]: Please delete this new line.

File: plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/EditMavenArtifactDialog.java [hunk 2]
```diff
@@ -188,10 +189,14 @@ private void createRawTab(@NotNull TabFolder folder) {
...
         gd.heightHint = UIUtils.getFontHeight(container.getFont()) * 12;
 
+        UIUtils.createInfoLabel(container, UIConnectionMessages.dialog_edit_driven_edit_maven_field_text_message);
         fieldText = new Text(container, SWT.V_SCROLL | SWT.MULTI | SWT.BORDER);
+
         fieldText.setLayoutData(gd);
         fieldText.addModifyListener(event -> parseArtifactText());
 
+
+
```
Comment (flambchap) [inline plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/EditMavenArtifactDialog.java]: Please delete these new lines.

**Comment by flambchap** [comment]
Could you please provide some screenshots with the new look?

**Comment by Destrolaric** [comment]
image
image

**Pull request merged**